interfaces/with-page-layout.ts

```diff
@@ -4,5 +4,4 @@ export type WithPageLayout<Q = {}> = NextPage<Q> & {
   PageLayout?: React.ComponentType<any>;
   SEO?: SEOobject;
   updateSEO?: (SEO: SEOobject) => void;
-  isPrivateRoute?: boolean;
```

Comment:
I removed all traces of `isPrivateRoute` which the `<PrivateWrapper />` component relied on.